frontend/packages/dev-console/src/components/HintBlock.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { PageHeading } from '@console/internal/components/utils';
+import SectionDivider from './import/section/FormSectionDivider';
+
+interface HintBlockProps {
+  title: string;
+  children?: React.ReactNode;
+}
+const HintBlock: React.FC<HintBlockProps> = (props) => (
+  <React.Fragment>
+    <PageHeading title={props.title}>{props.children}</PageHeading>
+    <SectionDivider />
```

Comment:
This is a very harsh line:
![image](https://user-images.githubusercontent.com/14068621/60319926-1e805c80-9946-11e9-8e4e-feb158812713.png)

UX design had a soft line:
![image](https://user-images.githubusercontent.com/14068621/60319936-29d38800-9946-11e9-9794-365e8405c38b.png)

Reply:
updated the border color